src/components/tabs/Tab.tsx

```diff
@@ -1,27 +1,31 @@
-import React from 'react'
-import clsx from 'clsx'
-import classes from './styles.module.css'
+import React from 'react';
```

Comment:
can you cross check your prettier config, this should not be adding formatting ';'